azure-pipelines.yml

```diff
@@ -183,7 +183,7 @@ stages:
             versionSpec: '$(python.version)'
           displayName: 'Use Python $(python.version)'
         - script: |
-            python -m pip install --upgrade build pip setuptools wheel
+            python -m pip install --upgrade build pip setuptools wheel "numpy>=2.0"
```

Comment:
I don't understand the need for the specification of `numpy` here, doing an installation with `pip` will always install the latest version so specifying just `numpy` should suffice for every use-case (i.e., for Python 3.8 the latest pre-2.0 version and for all others the latest available 2.0.x version

Reply:
OK, we can revert, I just wanted to be specific.  I don't think we need to worry about testing with both numpy 1.something and 2.something.   I was burned recently by my own misconfiguration of testing, and wanted "numpy 2, dammit" just to make sure it would pass... but we can revert this file.